src/components/NavbarMobile.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <el-collapse v-model="activeNames">
-  <el-collapse-item title="Menu" name="1">
+  <img id="img-navbar" src="../assets/openpromises.png" height="32px"/>
+
+  <el-collapse-item title="" name="1">
```

Comment:
ideally we want to have the hamburger icon, but for now, we retain the 'Menu' text so that it's clear to the user.